package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "openmct",
-  "version": "0.14.0-SNAPSHOT",
+  "version": "0.14.0",
```

Comment:
@akhenry Shouldn't this be v0.15.0?

Reply:
Last Open MCT release was [v0.13.3](https://github.com/nasa/openmct/releases/tag/v0.13.3). [The process](https://github.com/nasa/openmct/blob/v0.13.3/docs/src/process/version.md#version-numbering)  is that the version number is the next release with '-SNAPSHOT' appended to it. When that version is released, the `-SNAPSHOT` is removed, and the release is tagged. It is then updated with the next version number and a '-SNAPSHOT'.

Reply:
The reason I asked was the reference to 0.15.0 version number in the README. That file needs to be updated then.

Reply:
Good catch. Fixed in https://github.com/nasa/openmct/pull/2575